src/routes/+page.server.ts

```diff
@@ -1,15 +1,15 @@
 import * as prismic from "@prismicio/client";
 import type * as prismicT from "@prismicio/types";
+import type { PageServerLoad } from "./$types";
 
-export async function get({ fetch }) {
+export const load = (async ({ fetch }) => {
 	const endpoint = prismic.getEndpoint("svelte-package-dev");
 	const client = prismic.createClient(endpoint, { fetch });
 
 	const document: prismicT.PrismicDocument = await client.getSingle("homepage");
 
 	return {
-		body: {
-			document,
-		},
+		document,
 	};
-}
+// eslint-disable-next-line prettier/prettier
+}) satisfies PageServerLoad;
```

Comment:
`satisfies` is a TypeScript 4.9 feature and usually shouldn't be shipped in library code while adoption is low.

Since this is in `src/routes`, is it correct to say this code will not be included in the published package?

Reply:
@angeloashmore That is correct. This will not get shipped in the library.